spec/features/reviews/create_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe "As a visitor", type: :feature do
+  before(:each) do
+    @shelter = Shelter.create(name:    "Broomfield Pet Shelter",
+                             address: "1111 fake st.",
+                             city:    "Broomfield",
+                             state:   "CO",
+                             zip:     "80020")
+
+    visit "/shelters/#{@shelter.id}"
+  end
```

Comment:
Great use of `before(:each)`. We should try to use it in some of the existing tests, too.